apps/src/studio/levels.js

```diff
@@ -2977,16 +2977,14 @@ levels.js_hoc2015_change_setting = {
   ],
   'progressConditions' : [
     // Collected all the items and set the right properties?  Success.
-    { required: { 'setMap': true, 'setDroidSpeed': true, 'collectedItemsAtOrAbove': 3 },
+    { required: { 'setDroid': true, 'setDroidSpeed': true, 'collectedItemsAtOrAbove': 3 },
```

Comment:
Is there a `setDroid` option for `progressConditions`? I don't see it in `Studio.conditionSatisfied`

There is a `hasSetDroid` which is initialized to `false`, but never set.
There is a `hasSetSprite` which is never initialized/reset, but is set in `callCmd`

The droplet levels do call a `setDroid` api, which calls `queueCmd` as `setSprite`
The blockly levels always generate `setSprite` directly, bypassing the `setDroid` API.